spec/game_spec.rb

```diff
@@ -18,51 +18,149 @@ def print_message(string)
 end
 
 describe Game do
+  before(:each) do
+    @board = Board.new
+    @rules = Rules.new
+    @console = FakeConsole.new
+    @game = Game.new(@board, @console, @rules)
+  end
+
   describe 'welcome' do
     it 'prints a string to welcome user to the game' do
-      console = FakeConsole.new
-      board = Board.new
-      game = Game.new(board, console)
 
-      game.welcome
+      @game.welcome
 
-      expect(console.printed_string).to eq('Welcome to Tic Tac Toe')
+      expect(@console.printed_string).to eq('Welcome to Tic Tac Toe')
     end
   end
 
-  describe 'play' do
+  describe 'turn' do
     it 'marks a board appropriately with an X in the center' do
-      board = Board.new
-      console = FakeConsole.new
-      console.stub_get_input('5')
-
-      game = Game.new(board, console)
+      @console.stub_get_input(['5'])
 
-      game.play
+      @game.turn
 
       expected_output  = "   |   |  \n"
       expected_output += "-----------\n"
       expected_output += "   | X |  \n"
       expected_output += "-----------\n"
       expected_output += "   |   |  \n"
-      expect(console.print_message(console.printed_string)).to eq(expected_output)
+      expect(@console.print_message(@console.printed_string)).to eq(expected_output)
     end
 
     it 'marks a board appropriately with an X in the upper right corner' do
-      board = Board.new
-      console = FakeConsole.new
-      console.stub_get_input('3')
+      @console.stub_get_input(['3'])
 
-      game = Game.new(board, console)
-
-      game.play
+      @game.turn
 
       expected_output  = "   |   | X\n"
       expected_output += "-----------\n"
       expected_output += "   |   |  \n"
       expected_output += "-----------\n"
       expected_output += "   |   |  \n"
-      expect(console.print_message(console.printed_string)).to eq(expected_output)
+      expect(@console.print_message(@console.printed_string)).to eq(expected_output)
+    end
+  end
+
+  describe 'switch_players' do
+    it 'sets X as the current player when the game is started' do
+
+      expect(@game.switch_players).to eq('O')
+    end
+  end
+
+  describe 'play' do
+    it 'ends the game after X has a winning horizontal combination' do
+
+      @board.mark(0)
+      @board.mark(1)
+      @console.stub_get_input(['3'])
+      @game.play
+
+      expect(@rules.game_over?(@board)).to eq true
+    end
+  end
+
+  it 'ends the game after X has a winning diagonal combination' do
+
```

Comment:
I agree with @aaizuss above - get rid of the extra spaces between `it "does something" do` and  the first line of the test